# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../chrpath/chrpath-0.13-testsuite-1.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Submitted By: Mario Fetka (geos_one) (mario dot fetka at gmail dot com)
Date: 2010-04-04
Initial Package Version: 0.13
Origin: me
Upstream Status: unknown
Description: make the testsuite run with the new mltilib design
 
diff -Naur chrpath-0.13.orig/testsuite/runtest.sh chrpath-0.13/testsuite/runtest.sh
--- chrpath-0.13.orig/testsuite/runtest.sh	2010-04-04 09:02:21.859114159 +0000
+++ chrpath-0.13/testsuite/runtest.sh	2010-04-04 09:42:08.762112560 +0000
@@ -2,6 +2,8 @@
 
 set -x
 
+export LD_LIBRARY_PATH=$PWD/../.libs:$LD_LIBRARY_PATH
+
 CHRPATH=../chrpath
 
 retval=0
